python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -578,10 +577,6 @@ def get_vertex_data(self, vertex_ids=None, types=None, columns=None):
             if vertex_ids is not None:
                 if isinstance(vertex_ids, int):
                     vertex_ids = [vertex_ids]
-                elif not isinstance(
-                    vertex_ids, (list, slice, np.ndarray, self.__series_type)
-                ):
-                    vertex_ids = list(vertex_ids)
```

Comment:
Decided to let the underlying library handle it, we did not had `cp.array` here which lead to this slowdown.

Reply:
The combination base to test here is massive if want to handle this. 

Example, `dask_cp_array, dask_np_array, cudf.Series, dask_cudf.Series, cp.array` as an example of stuff that is  gonna be converted to list .